shared/localization/locales/ca.json

```diff
@@ -639,7 +639,7 @@
     "message": "Utilitza formats de vídeo per al contingut animat"
   },
   "core/audits/byte-efficiency/legacy-javascript.js | description": {
-    "message": "Els polyfills i les transformacions permeten als navegadors heretats fer servir funcions de JavaScript noves. No obstant això, molts no són necessaris per als navegadors moderns. Per al teu JavaScript empaquetat, adopta una estratègia de desenvolupament de scripts moderna utilitzant la detecció de la funció mòdul / no mòdul per reduir la quantitat de codi enviat als navegadors moderns, mantenint al mateix temps la compatibilitat amb els navegadors heretats. [Informació sobre com pots utilitzar el JavaScript modern](https://web.dev/publish-modern-javascript/)"
+    "message": "Els polyfills i les transformacions permeten als navegadors heretats fer servir funcions de JavaScript noves. No obstant això, molts no són necessaris per als navegadors moderns. Per al teu JavaScript empaquetat, adopta una estratègia de desenvolupament de scripts moderna utilitzant la detecció de la funció mòdul / no mòdul per reduir la quantitat de codi enviat als navegadors moderns, mantenint al mateix temps la compatibilitat amb els navegadors heretats. [Informació sobre com pots utilitzar el JavaScript modern](https://web.dev/articles/publish-modern-javascript)"
```

Comment:
i18n string changes need to go through the translator pipeline, but I imagine this would be the result anyway. Did you run `yarn update:sample-json`?

Reply:
I think reverting your changes makes sense. Most of these files (with the exception of en-US & en-XL) we get from the translator engine in g3.

After this PR lands we can upload the new i18n strings to g3. Then once the new strings go through the translator pipeline we will pull them in before the next release.

Reply:
So revert everything in `locales` directory but leave all the `.js` changes?